apps/guinea-pig-nestjs/src/app.service.ts

```diff
@@ -1,9 +1,14 @@
-import { Injectable } from '@nestjs/common';
-import { Result } from './app.controller';
+import { Injectable } from "@nestjs/common";
+import { Result } from "./app.controller";
```

Comment:
@GabiGrin , i think you have some autocorrect. Can you change it back please?

Reply:
done. was the default prettier config which is set to "singleCommas: true"

I also added minimal prettier config to that project to avoid that